src/components/home/StorySection/Story.css

```diff
@@ -13,75 +13,66 @@
     align-items: center;
     gap: 8px;
     align-self: stretch;
-
 }
 
+
 .our-story {
     margin: 0;
-    font-family: sans-serif;
-    font-style: normal;
-    font-weight: 800;
+    color: #18181A;
+    text-align: center;
+    font-family: Open Sans;
```

Comment:
can remove in the next PR, no longer need to set font-family unless the design font is not "Open Sans, san-serif"